### ISSUE TITLE
Subsonic API does not work without php-xml and php-dom installed

### ISSUE THREAD

**Issue by esalgado** [ampache/ampache#1749]
This is the second time this happens to me, but first time I solved it I did not take note of the steps needed to solve. In the help section to use Subsonic, it is not mentioned the need to have installed php-xml and php-dom in order to use the api. It is not installed by default and not listed as requisite in the Wiki and in test.php

Once I installed php56-xml and php56-dom, I could solve the issue, and get a response from /rest/ping.view

# Environment

 * Ampache version: 3.8.8
 * Web server + version: nginx 1.14.0_6,2
 * Server operating system: FreeBSD 11.1
 * Client operating system: Arch Linux, Firefox browser

 * Client type:
   - SubSonic

# Logs

ampache error.log:
2018/07/21 20:22:48 [error] 61134#100065: *9 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'DOMDocument' not found in /usr/local/www/music/lib/class/subsonic_api.class.php on line 190" while reading response header from upstream, client: 192.168.1.160, server: 192.168.1.161, request: "GET /rest/ping.view HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.1.161"

nginx access.log:
192.168.1.160 - - [21/Jul/2018:20:22:48 +0200] "GET /rest/ping.view HTTP/1.1" 500 5 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:61.0) Gecko/20100101 Firefox/61.0"
1

**Comment by wagnered** [comment]
@esalgado 
Thanks for the info.

Most installations are on Linux systems which have those specific modules built into the php executable.  Even when building from source, as I just did with php-7.2,  I did not need to specify those modules to be added: PHP packages for Windows also seem to be built with most commonly used modules.

Evidently FreeBSD, is a system  that doesn't do that.

I'll add them to the README.md page for the next commit.

Also the wiki can be edited by any member of github, So if you notice any peculiarities of Ampache on FreeBSD that might be of help to others, you are welcome to add them to the wiki.
